modules/nw-configuring-qinq-sriov-proc.adoc

```diff
@@ -0,0 +1,191 @@
+  resourceName: resource810
+----
+
+. Optional: Label the SR-IOV capable cluster nodes with `SriovNetworkNodePolicy.Spec.NodeSelector` if they are not already labeled. For more information about labeling nodes, see "Understanding how to update labels on nodes".
+
+. Create the `SriovNetworkNodePolicy` object:
++
+[source,terminal]
+----
+$ oc create -f <name>-sriov-node-network.yaml
+----
++
+After applying the configuration update, all the pods in `sriov-network-operator` namespace move to the `Running` status.
+
+. Create a `SriovNetwork` object, setting the outer VLAN referred to as the S-tag, or `Service Tag`, as it belongs to the infrastructure.
++
+[NOTE]
+====
+You need to configure this VLAN on the switch's trunk interface. Additionally, some switches may need further configuration to support QinQ tagging.
+====
+
+.. Using the following YAML example,create a file named `nad-sriovnetwork-1ad-810.yaml`:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriovnetwork-1ad-810
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: '{}'
+  vlan: 171 <1>
+  vlanProto: "802.1ad" <2>
+  networkNamespace: default
+  resourceName: resource810
+----
++
+<1> This sets the S-tag VLAN tag to 171. 
+<2> Specifies the VLAN protocol to assign to the VF. Allowed values are `802.1ad` and `802.1q`. The default is `802.1q`. 
+
+.. To create the object, enter the following command:
++
+[source,terminal]
+----
+$ oc create -f nad-sriovnetwork-1ad-810.yaml
+----
+
+. Create a Network Attachment Definition with an inner VLAN. The inner VLAN is often referred to as the C-tag, or `Customer Tag`, as it belongs to the Network Function.
+
+.. Using the following YAML example, create a file named `nad-cvlan100.yaml`:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: nad-cvlan100
+  namespace: default
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "vlan-100",
+      "plugins": [
+        {
+          "type": "vlan",
+          "master": "net1", <1>
+          "vlanId": 100,
+          "linkInContainer": true, 
+          "ipam": {"type": "static"}
+        }
+      ]
+    }
+----
++
+<1> Specifies the virtual function (VF) interface inside the pod. The default name is `net1` since the name is not set in the pod annotation. 
+
+.. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f nad-cvlan100.yaml
+----
+
+.Verification
+
+Verify QinQad is active on the node by following this procedure: 
+
+. Create an example test pod spec such as the following, and then save the YAML in the `test-qinq-pod.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: sriovnetwork-1ad-810, nad-cvlan100
+spec:
+  containers:
+    - name: test-container
+      image: quay.io/openshifttest/hello-openshift:openshift
```

Comment:
you need to use a image that support sriov? Please use an example of other pages.